### PR TITLE
Update Greenplum copyrights and legal statements Part I

### PR DESCRIPTION
**Pull request by zhangh43**
Copyrights in Greenplum source files are outdated and inconsistent.
Some of files are missing copyright statements while other files
are using older copyright statements, e.g. EMC and/or Pivotal.

This PR is the initial step to update the copyright statements.
All the `Pivotal (Software, Inc)` and its variants are replaced by `VMware, Inc. or
its affiliates`.

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -2050,8 +2050,8 @@ permitted provided that the following conditions are met:
       of conditions and the following disclaimer in the documentation and/or other materials
       provided with the distribution.
 
-THIS SOFTWARE IS PROVIDED BY TOMAS VONDRA, CONVERSANT INC, PIVOTAL SOFTWARE INC. AND ANY
-OTHER CONTRIBUTORS (THE "AUTHORS") ''AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+THIS SOFTWARE IS PROVIDED BY TOMAS VONDRA, CONVERSANT INC, VMWARE, INC. OF ITS AFFILIATE.
```
Comment (ashwinstar) [inline NOTICE:2053]: s/OF ITS AFFILIATE/OR ITS AFFILIATES

**Review by ashwinstar — COMMENTED** [review]
LGTM. Will let @kmacoskey provide the approval for the PR.

**Comment by kmacoskey** [comment]
Linking the two pull requests together, here is the accompanying changes to other copyright statements: https://github.com/greenplum-db/gpdb/pull/11258

**Review by kmacoskey — APPROVED** [review]
I have not reviewed each 1083 files individually. However a spot check of the various situations and conventions to me looks ok.

**Pull request merged**